src/expander.rs

```diff
@@ -335,14 +334,12 @@ where
     /// Writes the configuration register of the given bank
     fn write_conf(&mut self, bank: Bank) -> Result<(), <B as Write>::Error> {
         match bank {
-            Bank::Bank0 => self.bus.write(
-                self.address,
-                &[COMMAND_CONF_0, self.configuration_0.as_value().to_owned()],
-            ),
-            Bank::Bank1 => self.bus.write(
-                self.address,
-                &[COMMAND_CONF_1, self.configuration_1.as_value().to_owned()],
-            ),
+            Bank::Bank0 => self
+                .bus
+                .write(self.address, &[COMMAND_CONF_0, self.configuration_0.as_value().clone()]),
```

Comment:
As u8 implements the Copy trait, no cloning is required, as also suggested by Clippy.
Please try to dereference/copy instead:
```rust
*self.configuration_0.as_value()
```

This also applies to lines 342, 351, 354, 363 and 366.